scraper/scraper.js

```diff
@@ -112,8 +113,29 @@ async function loadBugs() {
     return JSON.stringify(Object.values(resultMap));
   });
 
-  await browser.close();
   const arr = JSON.parse(result);
+
+  for (bug of arr) {
+    await page.goto(`${RARITY_BASE_URL}${encodeURIComponent(bug.name.replace(' ', '_'))}`);
+    bug.rarity = await page.evaluate(() => {
+      const rarity = document.querySelector("[data-source=rarity] div").textContent;
+      if (rarity.includes("Common")) {
```

Comment:
I'm fine with keeping these values as either the names they are now or a number value between 1 -> 5. When displaying these we are probably just going to show either the text or if we are fancy a few star icons. I think we should start with just the words